### ISSUE TITLE
Edit log entries in my machine's timezone

### ISSUE THREAD

**Issue by prismec** [kriskbx/gitlab-time-tracker#45]
```
gtt start 4711
gtt stop 4711
gtt edit L74n5Rn2X
```

displays 

```
{
	...,
	"start": "2018-05-09T06:39:06.523Z",
	"stop": "2018-05-09T06:39:11.235Z"
}
```

It would be great if the timestamps are in my local timezone or if I could configure that somehow.

**Comment by kriskbx** [comment]
I will implement timezones in the next version. You can specify them then in your config via `timezone: "Europe/Berlin"`. I'll keep you up to date when it goes live.

But keep in mind, there's a big caveat: As explained in #39 gtt doesn't sync the date to GitLab (because there's no way to do this via an API call at the moment). The date of your time record on GitLab matches the current date when you ran the `sync` command and it matches GitLabs timezone settings. You're local timestamps stay local and are only used to calculate the duration at the moment (and for the `log` command).

**Comment by kriskbx** [comment]
I just released 1.7.5 which includes the ability to specify timezones in your config file.
